README.md

```diff
@@ -762,10 +763,10 @@ func printSomeFile() {
 
 Instead, we should be using Swift's `try`/`catch` behavior when it is appropriate to know the reason for the failure.
 
-Use a `struct` such as the following:
+You can use a `struct` such as the following:
 
 ```swift
-struct Error: ErrorType {
+struct Error: Swift.Error {
```

Comment:
I was struggling to come up with a good name for this, since `Error` works so well and anything else just seems extraneous. I wish Apple had kept this as `ErrorType` or `ErrorProtocol` =\.

This works just fine if you just use `Swift.Error` here instead however, and it lets you keep the good naming. None of the alternatives that I've considered (`ErrorWithMessage`, `DescriptiveError`, `ErrorContainer`, `Errrrroooorrrr`, etc.) seem to be as clean as just `Error`.

This is especially useful when you want to namespace an error type within a class. For example:

```
class FileLoader {

    enum Error: Swift.Error {
        // ...
    }

    // V.S.

    enum FileLoaderError: Error {
        // ...
    }

}
```

`FileLoader.Error` is a far better than `FileLoader.FileLoaderError` and fits more with the Swift API Guideline of avoiding needless words.